### ISSUE TITLE
Fix examples on https://jeremylong.github.io/DependencyCheck/general/suppression.html

### ISSUE THREAD

**Issue by kwin** [dependency-check/DependencyCheck#1940]
**Describe the bug**
The example XMLs in the aforementioned page don't reference the suppression XSD correctly.

**Expected behavior**
The XSD should be used by XML editors to validate the given schema. For that the XSD needs to be referenced correctly. An example of a correct reference can be found in https://www.ibm.com/developerworks/library/x-tipsch/index.html#gettingdown.

**Additional context**
Basically each schema is referenced via a namespace. The example XMLs https://jeremylong.github.io/DependencyCheck/general/suppression.html currently only define a namespace but not the underlying schema.

**Comment by aikebah** [comment]
There is no need to include the XMLSchena instance. The schemaLocation attribute of the http://www.w3.org/2001/XMLSchema-instance namespace is only a hint for xml validators on where it might resolve a copy of the schema definition for that namespace.

One of the issues I've seen with IDEs and other XML validation tools is that they take it as the location to use rather than a hint and override my own local schema catalog of xml-namespace-URI => xsd-file-location. I like to keep my XSDs locally cached and registered in the XML Catalog for my tools if they are important to me; validation should run fine when disconnected from the web.

**Comment by kwin** [comment]
IMHO if the hint by default is a URL all XML editors will use that by default. You can still override the catalog if you want to to make the validation work offline.

**Comment by aikebah** [comment]
Sure, but my point is: the schemaLocation hint is just that: a hint, so the examples as is are perfectly fine XML as they are. Just voicing my view on the topic that this is not a bug. The schemaLocation in my view is just a bunch of nearly useless bytes repeated in every XML document that can be perfectly replaced by a proper XML catalogue in your tool of choice towards the same effect: in-tool validation of the XML.

**Comment by kwin** [comment]
Yes, it is just a hint, but it is further defined within https://www.w3.org/TR/xmlschema11-1/#schema-loc. So I think it makes sense to by default give a URL to ease validation.

**Comment by jeremylong** [comment]
I agree with @aikebah; and in reality just using what is in the current suppression example most IDEs I use ask if it should download the XSD and perform proper validation/inteli-sense, etc. However, @kwin you would rather see:

```xml
<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3"
 xsi:schemaLocation="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3
                     https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
```

**Comment by SachinAditya** [comment]
@kwin 
Hi, I’d like to work on this issue if it’s still available 🙂

**Comment by chadlwilson** [comment]
This is somewhat duplicated by the discussion at #7627 so please read the discussion there, but updating the examples to have a default schema location (and ensuring they work) is probably reasonable I think.
